src/examples/table/TableDraggable.tsx

```diff
@@ -104,7 +104,7 @@ class DraggableCell extends React.Component<IDraggableCellProps> {
   }
```

Comment:
The use of `primaryHue` for the draggable grip feels like it needs some design review. I'm sure they'll come across it but just wanted to leave a comment about it here for posterity. 🧐

Reply:
I suspect they'll suggest we update it to `foreground.subtle` like in `Draggable.Grip`, but maybe there's a reason it's blue here.

Reply:
Agreed. I don't think there is substantial design reasoning behind the primary blue. It was originally applied as a one-off in this example. The Garden designed `Draggable.Grip` should be considered to be the guiding principle.